plugins/wazuh_check_updates/public/hooks/useAvailableUpdates.ts

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import { AvailableUpdates } from '../../common/types';
+import { routes } from '../../common/index';
+import { getCore } from '../plugin-services';
+
+export const useAvailableUpdates = () => {
+  const [availableUpdates, setAvailableUpdates] = useState<AvailableUpdates>({
+    mayor: [],
+    minor: [],
+    patch: [],
+  });
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  const refreshAvailableUpdates = (forceUpdate = false) => {
+    (async () => {
+      try {
+        setIsLoading(true);
+        const response = await getCore().http.get(`${routes.checkUpdates}`, {
+          query: {
+            checkAvailableUpdates: forceUpdate,
+          },
+        });
+        setAvailableUpdates(response);
+        setError(undefined);
+      } catch (error: any) {
+        setError(error);
```

Comment:
**question**: if there is an error, should the available updates be cleaned?

Reply:
Due to an error, the value of available updates was cleared